Software/spartan/src/sensors/dps422xtsa1.cpp

```diff
@@ -0,0 +1,41 @@
+#include "dps422xtsa1.h"
+
+spartan::DPS422XTSA1::DPS422XTSA1(int bus, uint8_t address) 
+	: Sensor(bus, address), m_i2c(bus, true)
+{
+	// Implementation
+}
+
+const char* spartan::DPS422XTSA1::name() const
+{
+	// Implementation
+	const char* sensor_name = "DPS422XTSA1";
+	return sensor_name;
```

Comment:
We could probably just directly return the string literal "DPS422XTSA1" (not too big of a deal, but avoids unnecessary code).